management-service/services/segmenter_service_test.go

```diff
@@ -1176,7 +1175,7 @@ func NewCountrySegmenter(_ json.RawMessage) (segmenters.Segmenter, error) {
 	return segmenters.NewBaseSegmenter(countryConfig), nil
 }
 
-func NewAreaSegmenter(configData json.RawMessage) (segmenters.Segmenter, error) {
+func NewAreaSegmenter(_ json.RawMessage) (segmenters.Segmenter, error) {
```

Comment:
Minor refactoring of the method argument because it isn't even used at all in the function (maybe we can even simplify the signature?)